### PR TITLE
Added meta license

### PR DESCRIPTION
**Pull request by willdurand**
Hi,

That's allow to add a `Resources/meta/LICENSE` file by using the `licensify` command.

Regards,
William.

### CODE REVIEW

File: Command/LicensifyCommand.php [hunk 1]
```diff
@@ -68,5 +69,17 @@ class LicensifyCommand extends ContainerAwareCommand
...
+        $metaPath   = $bundle->getPath() . '/Resources/meta/';
+        $metaFile   = $metaPath . '/LICENSE';
+
+        if (! is_dir($metaPath)) {
+            $filesystem = new Filesystem();
+            $filesystem->mkdir($metaPath);
+            $filesystem->touch($metaFile);
+        }
+
+        file_put_contents($metaFile, $license);
```
Comment (schmittjoh) [inline Command/LicensifyCommand.php:83]: How about adding a file_exists check just to be save?

Comment (willdurand) [inline Command/LicensifyCommand.php:83]: Yep, with an output message ?

Comment (schmittjoh) [inline Command/LicensifyCommand.php:83]: Yep, sounds good.

**Comment by willdurand** [comment]
Updated with the `file_exists` check.

**Comment by schmittjoh** [comment]
Thanks, I have changed the check a bit, I think we were thinking in different directions :)

**Pull request merged**